### PR TITLE
Add core-types readme

### PR DESCRIPTION
**Pull request by jkoenig134**
# Readiness checklist

-   [ ] I added/updated tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

### CODE REVIEW

File: packages/core-types/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Core Types
+
+[![npm version](https://badge.fury.io/js/@nmshd%2fcore-types.svg)](https://www.npmjs.com/package/@nmshd/core-types)
+
+The core-types library provides common types for the transport and content library.
+
+## Documentation
+
+You can find the documentation for Enmeshed on [enmeshed.eu](https://enmeshed.eu).
```
Comment (Milena-Czierlinski) [inline packages/core-types/README.md:9]: enmeshed not capitalized 🤔 (also below)

File: README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ This monorepo consolidates the following packages:
 
 | Package                              | Version                                                                                                           |
 | :----------------------------------- | :---------------------------------------------------------------------------------------------------------------- |
+| [core-types](packages/core-types/)   | [![version](https://badge.fury.io/js/@nmshd%2fcore-types.svg)](https://www.npmjs.com/package/@nmshd/core-types)   |
```
Comment (Milena-Czierlinski) [inline README.md:9]: Should we really keep the version? It should be the same for all libraries, so there is no actual information that this column adds.

File: packages/core-types/README.md (hunk 1 above)
Comment (jkoenig134) [reply packages/core-types/README.md:9]: copy paste from other places.. but ok

File: README.md (hunk 2 above)
Comment (jkoenig134) [reply README.md:9]: it looks shitty without. so yes.

**Review by Milena-Czierlinski — APPROVED** [review]

**Pull request merged**